### DISCUSSION TITLE
Inputs getting squashed

### DISCUSSION THREAD

**Discussion by SpasiboKojima** [saadeghi/daisyui#3132] [Q&A] [answered]
UPD; Fixed by using zoom with ctrl + back and forth, now it doesn't come back even after browser relaunch. 🧐

Noticed this happening sometimes on some container sizes, they shift when I increase container width by 1 pixel and they aren't even the same being aligned block-wise. No style rewrites, pure DaisyUI.
Happens not only with radios, but with toggles and checkboxes as well.
Only on Firefox, Chrome is fine.
Doesn't seem to be from flex as `flex-shrink: 0;` doesn't do anything.
Basically my components:
```
<div className="flex flex-col gap-4">
  <label className="flex cursor-pointer items-center gap-2">
    <input name="vital" type="radio" className="radio checked:bg-blue-500 radio-xs" value="true" />
    <span>Yes</span>
  </label>
  <label className="flex cursor-pointer items-center gap-2">
    <input name="vital" type="radio" className="radio checked:bg-blue-500 radio-xs" value="false" />
    <span>No</span>
  </label>
</div>
```

https://github.com/user-attachments/assets/2fe0e844-0157-40bb-a31f-056f6e97b193

Anyone has any idea what's causing this?

**Comment by saadeghi** [comment]
I don't see the issue.  
- Are you using the latest version of daisyUI?
- Which version of Firefox has this issue?
- Can you share a [Tailwind Play](https://daisyui.com/tailwindplay/) link?

**Reply by SpasiboKojima** [reply, marked as answer]
Does the video not work for you?
- Sure, I'm on the latest version of everything (for what it's worth, DaisyUI, Tailwind, React and Vite);
- 127.0.2;
- I can't reproduce it anymore and I assume it was OS/browser thing (I'm on Windows 11 WSL), so it's all good, thank you.

**Reply by saadeghi** [reply]
>Does the video not work for you?

Yeah I see the video but when I run your code example, I don't see the issue.  

>I can't reproduce it anymore and I assume it was OS/browser thing (I'm on Windows 11 WSL), so it's all good, thank you.

Maybe it was a temporary bug on a specific version of Browser and it's fixed now.  
Or maybe it was using an older version of daisyUI. I remember we had this bug (probably a year ago If I remember correctly) and it was a flex shrink issue which was fixed by adding `flex-shrink: 0`

Anyway. I'm glad the issue is gone.
